docs/Components/Page.md

```diff
@@ -19,27 +21,30 @@ render(ExtensionPoint.MyExtension, (root) => {
 });
 ```
 
-#### React
+#### React example
 
 ```jsx
-import {render, ExtensionPoint, Page} from '@shopify/argo-admin-react';
+import {extend, render, ExtensionPoint, Page} from '@shopify/argo-admin-react';
 
 function App() {
   return <Page>This is the best extension.</Page>;
 }
 
-render(ExtensionPoint.MyExtension, () => <App />);
+extend(
+  'Playground',
+  render(() => <App />),
+);
 ```
 
 ## Props API
 
-| Name             | Type             | Description                                                                               | Required |
-| ---------------- | ---------------- | ----------------------------------------------------------------------------------------- | -------- |
-| title            | `string`         | Page title, in large type.                                                                | ☑️       |
-| subTitle         | `string`         | Page subtitle, in regular type.                                                           |          |
-| primaryAction    | `Action`         | Primary page-level action. See [Action](#Action) action.                                  |          |
-| secondaryActions | `Action[]`       | Collection of secondary page-level actions. See [Action](#Action)actions.                 |          |
-| thumbnail        | `ThumbnailProps` | Thumbnail that precedes the title. See [Thumbnail Props API](./Thumbnail.md#Props%20API). |          |
+| Name             | Type             | Description                                                                              | Required |
+| ---------------- | ---------------- | ---------------------------------------------------------------------------------------- | -------- |
+| title            | `string`         | Page title, in large type                                                                | ☑️       |
```

Comment:
Nit: do we want to be consistent about using period (`.`) in props descriptions?

Reply:
I have no idea xD